lib/ui/home/home_screen.dart

```diff
@@ -103,38 +78,45 @@ class _HomeScreenState extends ConsumerState<HomeScreen> {
             },
           ),
           SafeArea(
-            child: Consumer(
-              builder: (context, widgetRef, _) {
-                return HomeHeader(
-                  // TODO: Move logic to VM on integration task
-                  date: DateTime.now().getFormattedString(),
-                  avatar: widgetRef.watch(_profileAvatarProvider),
-                );
-              },
+            child: HomeHeader(
+              date: today,
+              avatar: profileAvatar,
             ),
           ),
-          Positioned(
-            bottom: 206,
-            child: PageViewDotIndicator(
-              currentItem: _currentIndex,
-              count: mockSurveys.length,
-              alignment: Alignment.centerLeft,
-              padding: const EdgeInsets.symmetric(horizontal: space15),
-              margin: const EdgeInsets.symmetric(horizontal: space5),
-              size: const Size(
-                pagerIndicatorSize,
-                pagerIndicatorSize,
+          if (surveys.isNotEmpty)
+            Positioned(
```

Comment:
OPTIONAL: We could extract to new `_build` method to build the indicator widget.